cli/testflinger_cli/__init__.py

```diff
@@ -351,6 +414,32 @@ def submit_job_data(self, data):
             ) from exc
         return job_id
 
+    def submit_job_attachments(self, job_id: str, path: Path):
+        """Submit attachments archive for a job to the server
+
+        :param job_id:
+            ID for the test job
+        :param path:
+            The path to the attachment archive
+        """
+        try:
+            self.client.post_attachment(job_id, path)
+        except client.HTTPError as exc:
+            if exc.status == 400:
+                raise SystemExit(
+                    f"Unable to submit attachment archive for {job_id}"
+                ) from exc
+            if exc.status == 404:
+                raise SystemExit(
+                    "Received 404 error from server. Are you "
+                    "sure this is a testflinger server?"
+                ) from exc
+            # This shouldn't happen, so let's get more information
+            raise SystemExit(
+                "Unexpected error status from testflinger "
+                "server: {}".format(exc.status)
```

Comment:
As I understand from the docs, SystemExit should be raised with `sys.exit` 

https://docs.python.org/3/library/exceptions.html#SystemExit

Reply:
@plars Any thoughts on this? `SystemExit` is used throughout `testflinger_cli`, whereas `exit` is used throughout `testflinger_cli.client`. I followed the existing approach within each module but @thp-canonical made a similar comment regarding consistency. We could open a separate issue for this but I don't feel that addressing this is within the scope of the current PR.

Reply:
I suggest using sys.exit still, since you can also specify the exit code there

Reply:
@nadzyah I have now used `sys.exit` in the Testflinger methods affected by this PR but there are dozen more instances of this throughout Testflinger that are out of scope.